manifests/params.pp

```diff
@@ -38,6 +38,19 @@
             }
           }
         }
+        'Debian': {
+          if versioncmp($facts['os']['release']['major'], '7') > 0 {
```

Comment:
An alternative is ```if $facts['service_provider'] == 'systemd' {``` (see https://github.com/puppetlabs/puppetlabs-stdlib#service_provider) which could unifiy the Ubuntu and Debian code. At the same time Ubuntu would be make future proof.

Reply:
Also it's possible that debian 8 still runs on sysv